src/lib.rs

```diff
@@ -466,12 +466,17 @@ pub fn initialize() {
             InstructionSet::X86 => {
                 set_dispatch_table!(GLOBAL_STATE.converters, x86);
             }
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
Can you explain why is this needed? If target_arch is aarch64 is there a chance that an instruction set other than x86 is selected?

Reply:
When arch is not x86, the sse2 and avx2 optimized functions are not available, and compilation will fail:
```
   Compiling dcv-color-primitives v0.1.10 (/home/ubuntu/dcv-color-primitives)
error[E0433]: failed to resolve: could not find `sse2` in `convert_image`
   --> src/lib.rs:471:62
    |
471 |                 set_dispatch_table!(GLOBAL_STATE.converters, sse2);
    |                                                              ^^^^ could not find `sse2` in `convert_image`

error[E0433]: failed to resolve: could not find `avx2` in `convert_image`
   --> src/lib.rs:475:62
    |
475 |                 set_dispatch_table!(GLOBAL_STATE.converters, avx2);
    |                                                              ^^^^ could not find `avx2` in `convert_image`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0433`.
error: could not compile `dcv-color-primitives`.
```